### ISSUE TITLE
Add documentation for training

### ISSUE THREAD

**Issue by bakhtos** [bakhtos/panns_SED#1]
- Update the parameter descriptions
- Explain the loss through weak labels or strong labels
- Explain finetuning through checkpoint
